src/pages/home/sidebar/SidebarScreen/BaseSidebarScreen.js

```diff
@@ -62,10 +63,51 @@ class BaseSidebarScreen extends Component {
         Welcome.show({routes, showCreateMenu: this.showCreateMenu});
     }
 
+    componentDidUpdate(prevProps) {
+        if (!this.didBecomeInactive(prevProps)) {
+            return;
+        }
+
+        // Hide menu manually when other pages are opened using shortcut key
+        this.hideCreateMenu();
+    }
+
+    didBecomeInactive(prevProps) {
+        // When Report page is just opened
+        if (!this.props.isDrawerOpen && prevProps.isDrawerOpen) {
+            return true;
+        }
+
+        // When other page is just opened
+        if (!this.props.isFocused && prevProps.isFocused) {
+            return true;
+        }
+
+        return false;
+    }
+
+    isInactive() {
+        // When drawer is closed and Report page is open
```

Comment:
What is difference between didBecomeInactive and this funciton. Why can't we use didBecomeInactive?

Reply:
Add JSDocs for both functions with details.

Reply:
@aimane-chnaif Can you please do this?

Reply:
Comments for functions. There is no details about these functions and how they are used.